validator-node-setup.md

```diff
@@ -0,0 +1,157 @@
+# 1. Running a Node
+Here we describe how to run and sync a node using celestia-appd.
+## 1.2 Installing Dependencies
+First, make sure to update and upgrade the OS:
+```sh
+sudo apt update && sudo apt upgrade -y
+```
+Install essential packages:
+```sh
+sudo apt install curl tar wget clang pkg-config libssl-dev jq build-essential git make ncdu -y
+```
+Install GO:
```

Comment:
This deserves a chapter of its own 
```suggestion
## 1.2 Install GO:
```

Reply:
Separated the Go installation in a new chapter and added some more details about the packages installed.